Makefile

```diff
@@ -253,7 +265,7 @@ $(QEMU_PATH)/config-host.mak:
 # Need a PHONY target here, otherwise it mixes it with the folder name "qemu".
 .PHONY: qemu
 qemu: $(QEMU_PATH)/config-host.mak
-	make -C $(QEMU_PATH)
+	+make -C $(QEMU_PATH)
```

Comment:
This should have been a `$(MAKE)` to start with (and not `make`). I think that would sort out the parallelism if the top-level (this) make file was invoked with a `-j` flag.

Reply:
Ahh.  That makes sense.  It was on my TODO list to figure out why qemu was different.

Reply:
If you could update the patch for that and do a squash, force push to the same branch. Then I'd be happy to just merge this patch series as it is.